events/custom_test.go

```diff
@@ -12,10 +12,11 @@ import (
 func TestRollupRoundTrip(t *testing.T) {
 	data := MakeCustom(3)
```

Comment:
Shouldn't this be 4? Guess it doesn't matter since Add does an append...

Reply:
yeah, it's just an awkward (probably premature) optimization.